lib/complex.ex

```diff
@@ -1042,12 +1042,15 @@ defmodule Complex do
       true ->
         rho = abs(x)
         theta = phase(x)
-        s = multiply(power(rho, y.re), exp(multiply(negate(y.im), theta)))
+        s = multiply(pow(rho, y.re), exp(multiply(negate(y.im), theta)))
         r = add(multiply(y.re, theta), multiply(y.im, ln(rho)))
         new(multiply(s, cos(r)), multiply(s, sin(r)))
     end
   end
 
+  @deprecated "Use pow/2 instead"
+  def power(x, y), do: pow(x, y)
+
```

Comment:
@josevalim, Does this message look good? I directly copy-pasted it from https://elixir-lang.org/blog/2018/01/17/elixir-v1-6-0-released/

Reply:
Yup, thanks!